files/build_templates/docker_image_ctl.j2

```diff
@@ -685,6 +703,7 @@ start() {
         -e SX_API_SOCKET_FILE=/var/run/sx_sdk/sx_api.sock \
 {%- elif docker_container_name == "pmon" %}
         -v /var/run/hw-management:/var/run/hw-management:rw \
+        -v /etc/hw-management-sensors:/etc/hw-management-sensors:rw \
```

Comment:
please check if we can remove the mount: 
1. copy the file instead of symlink
2. you can keep with the symlink, but I think we can have read-only mount, and not read-write.